src/AdapterPluginManager.php

```diff
@@ -45,7 +58,7 @@ class AdapterPluginManager extends AbstractPluginManager
      * @return void
      * @throws Exception\RuntimeException if invalid
      */
-    public function validatePlugin($plugin)
+    public function validate($plugin)
```

Comment:
If you define the protected class property `$instanceOf` and assign it the value of `Adapter\AdapterInterface::class`, you can get rid of this method entirely. :smile: